<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPutar</class>
 <widget class="QWidget" name="SetPutar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="SETWAKTU">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-style: none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSpinBox" name="SETJAM_2">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>87</y>
      <width>267</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="SETJAM">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>12</y>
      <width>208</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>SET JAM</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="SETWAKTU_3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>312</y>
      <width>267</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="SETWAKTU_4">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>237</y>
      <width>249</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>SET MENIT</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BACK_2">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>479</y>
      <width>375</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>media/Frame 4.png</normaloff>media/Frame 4.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>375</width>
      <height>70</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DisplaySuhu">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>0</y>
     <width>512</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-style: none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="TERMO">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>170</y>
      <width>100</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>media/TERMO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BACK">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>479</y>
      <width>375</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>media/Frame 3.png</normaloff>media/Frame 3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>375</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="JUDUL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>11</y>
      <width>464</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap>media/JUDUL SET.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SUHU">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>215</y>
      <width>230</width>
      <height>234</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>140</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="DERAJAD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>40</width>
      <height>82</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>o</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>514</x>
     <y>21</y>
     <width>535</width>
     <height>507</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>media/TELKOM2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>Logo</zorder>
  <zorder>SETWAKTU</zorder>
  <zorder>DisplaySuhu</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
